### PR TITLE
add possibility to configure docker namespace for snapshots

### PR DESCRIPTION
**Pull request by riuvshin**
### What does this PR do?
add possibility to configure docker namespace for pushing snapshots

#### Changelog
add possibility to configure docker namespace for pushing snapshots

### CODE REVIEW

**Comment by garagatyi** [comment]
I think it is a bit confusing to call it docker namespace. Since it will go into changelog can you rephrase it.

**Comment by richtera** [comment]
:+1:
